examples/nips17_adversarial_competition/dataset/README.md

```diff
@@ -57,6 +57,12 @@ OUTPUT_DIR=images
 python download_images.py --input_file=${CSV_FILE} --output_dir=${OUTPUT_DIR}
 ```
 
+To accelerate the image downloading, `download_images_multiprocessed.py` spawns multiple processes.
```

Comment:
You're using multiprocessing.dummy module, which is a wrapper around threading module
(see [python2 doc](https://docs.python.org/2/library/multiprocessing.html#module-multiprocessing.dummy) and [python3 doc](https://docs.python.org/3/library/multiprocessing.html#module-multiprocessing.dummy)). So your program actually doing multi-threading and do not spawn multiple processes.

Could you please update description here as well as the name of the file to reflect that it's multi-threading.